### PR TITLE
restyle landing to look good on mobile

### PR DESCRIPTION
**Pull request by tehnalogos**
Screenshot 2024-04-18 at 9 31 52 AM
Screenshot 2024-04-18 at 9 32 15 AM
Screenshot 2024-04-18 at 9 33 26 AM
Screenshot 2024-04-18 at 9 33 41 AM

### CODE REVIEW

File: frontend/contracts/common.ts
Comment (tehnalogos) [inline frontend/contracts/common.ts:1]: lint only. sorry

File: frontend/contracts/factories/LSP1GraveForwarder__factory.ts
Comment (tehnalogos) [inline frontend/contracts/factories/LSP1GraveForwarder__factory.ts:1]: lint only. sorry

File: frontend/contracts/factories/index.ts
Comment (tehnalogos) [inline frontend/contracts/factories/index.ts:1]: lint only. sorry

File: frontend/contracts/index.ts
Comment (tehnalogos) [inline frontend/contracts/index.ts:1]: lint only. sorry

File: frontend/contracts/LSP1GraveForwarder.ts
Comment (tehnalogos) [inline frontend/contracts/LSP1GraveForwarder.ts:1]: lint only. sorry

**Review by democrazyfi — APPROVED** [review]

**Pull request merged**